docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -854,11 +1001,15 @@ Required ECS fields:::
 [[security-network-jobs]]
 == Security: Network
 
+Detect anomalous network activity in your ECS-compatible network logs.
+
 These configurations are only available if data exists that matches the 
 recognizer query specified in the
 https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_network/manifest.json[manifest file].
 
-Detect anomalous network activity in your ECS-compatible network logs.
+If you create these jobs in the *{ml-app}* app, you must first create the
```

Comment:
As above, this is not correct for the ML app - we run the query in the `manifest.json` against the select index pattern / data view.

Reply:
Ditto re changes in 33d2596b3801340540626e42ecab5a4bd5748e7c

Reply:
This change stating how you create the jobs in the ML app LGTM